### PR TITLE
Build macOS as universal

### PR DESCRIPTION
**Pull request by kubaau**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -31,13 +31,10 @@ jobs:
           asset_name: "cppsdk-${{ env.CPP_SDK_TAG }}-${{ env.BUILD_PLATFORM }}.zip"
```
Comment (djova-dolby) [inline .github/workflows/build.yml:31]: Can't comment on the env directly but it should become this right?
```
env:
      BUILD_PLATFORM: 'macos64-universal'
```

Comment (djova-dolby) [reply .github/workflows/build.yml:31]: or just append `-universal` after teh ${{ env.BUILD_PLATFORM }}

Comment (kubaau) [reply .github/workflows/build.yml:31]: I'll take a look at this when SDK is available.

Comment (djova-dolby) [reply .github/workflows/build.yml:31]: fair point.

**Pull request merged**